src/components/App/App.js

```diff
@@ -10,10 +10,10 @@ class App extends Component {
   };
 
   componentDidMount(){
-    this.makeFiveColors()
+    this.makeColors()
```

Comment:
I this this is the best naming convention for this function.

Reply:
I renamed it makeColor because we may modify it only make four or three etc